Hue.xcodeproj/project.pbxproj

```diff
@@ -611,6 +618,8 @@
 				PRODUCT_NAME = Hue;
 				SDKROOT = macosx;
 				SKIP_INSTALL = YES;
+				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
```

Comment:
Yeah, the migration process is a pretty invasive approach, but I can't complain, they sure show you all the changes (except when they update the Project's setting, including this very change)